src/clientManager.js

```diff
@@ -20,6 +20,19 @@ ClientManager.prototype.findClient = function(name) {
 	return false;
 };
 
+ClientManager.prototype.writeUser = function(userDict) {
+	var users_path = Helper.HOME + "/users" ;
+	var user_path = users_path + "/" + userDict.user + ".json";
+
+	mkdirp(users_path);
+	fs.writeFileSync(
+		user_path,
+		JSON.stringify(userDict, null, "  ")
+	);
+	// No effect on Windows (and no easy way to do something equivalent)
+	fs.chmodSync(user_path, "0660");
```

Comment:
Nice catch. This is even worse. Big :-1: because of that. This is way too intrusive. Setting "sane" defaults is a thing, but flat out overriding what the user has set is very hostile.